### PR TITLE
[master] resume

### PR DESCRIPTION
**Pull request by jamesli9572**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql/README.md) In-progress
-2. [Core Java Apps](./core_java/README.md) In-progress
-3. [Springboot Trading REST API](./springboot/README.md) In-progress
+# James Li . Jarvis Consulting
```
Comment (jarviscanada) [inline README.md:1]: run it through Grammarly

File: README.md [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+- **[Core Java Apps](./core_java)**: This project consist of three part
+    - The Java Grep App is a text search application. The application can search for a text pattern recursively in a given directory, and output matched lines to a file.
+    - The JDBC App is a data processing application which allows user to connect to PostgreSQL database and perfrom actions to the tables.
+    - The Java Twitter App is an command line application that allows user to Create, Read and Delete Tweets. It connects to Twitter REST API using authentication tool OAuth and perform part of CRUD implementation. 
+- **[SpringBoot App](./springboot)**: This Java application is an REST API designed for online stock trading simulation. The application allows a trader to sell and buy stock.
+- **[Hadoop](./hadoop)**: The Hadoop project is a data analysis project performed using Spark through Zeppelin Notebook. The Hadoop cluster was hosted on Google Cloud.
+- **[Spark/Scala](./spark)**:  In-Progress
+- **[Cloud & DevOps](./cloud_devops)**: Not started
```
Comment (jarviscanada) [inline README.md]: use action verb to start each description

please double-check the instruction

https://www.notion.so/jarviscanada/James-Resume-Workshop-418f43826b4e455e8b14ab640c2cd2a3#3e1a9f50eb6f49fc9440183286f3d7c6

**Review by jarviscanada — CHANGES_REQUESTED** [review]
fix and request another review from your 360 page

File: README.md (hunk 2 above)
Comment (jamesli9572) [reply README.md]: fixed the issue addressed, ready for review

File: README.md [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+
+
+- **[Cluster Monitor](./linux_sql)**: Implemented a Cluster Monitoring Solution to collect the hardware specifications of each node and monitor node resource usages. The data will be collected and stored in an RDBMS database.
+- **[Core Java Apps](./core_java)**: This project consists of three-part
+    - Developed a text search application. The application can search for a text pattern recursively in a given directory, and output matched lines to a file.
+    - Implemented is a data processing application which allows user to connect to the PostgreSQL database and perform actions to the tables.
+    - Designed a command-line application that allows users to Create, Read, and Delete Tweets. It connects to Twitter REST API using the authentication tool OAuth and performs part of CRUD implementation.
+- **[SpringBoot App](./springboot)**: Programmed a REST API designed for online stock trading simulation. The application allows a trader to sell and buy stock.
+- **[Hadoop](./hadoop)**: Implemented a data analysis project performed using Spark through Zeppelin Notebook. The Hadoop cluster was hosted on Google Cloud.
```
Comment (jarviscanada) [inline README.md]: project description is too short.. talk more about design pattern, libraries, testing, etc..

File: README.md [hunk 4]
```diff
@@ -1,4 +1,49 @@
...
+- **[Spark/Scala](./spark)**:  In-Progress
+- **[Cloud & DevOps](./cloud_devops)**: Not started
+
+## Professional Experiences
+
+**Software Developer,  Jarvis, Toronto (2019-Present):** Completing a comprehensive training bootcamp on core BigData technologies by industry experts and working on projects that include Cloud, DevOps, Java, Hadoop/Spark. Engage with my scrum team in the capacity of the scrum master to improve team cohesion and successfully pass team code reviews. Building Java micro-services using Core Java, Spring Framework, and Java development tool.
```
Comment (jarviscanada) [inline README.md]: please make it sounds like a professional experience rather than training.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,49 @@
...
+
+**Consultant, Delbridge Solutions, Vaugh (2017-2018):** Successfully implemented Sales Performance Management (SPM) solution for clients to manage compensations. Designed and configured SPM solutions according to client requirements within tight deadlines. Gathered and analyzed client requirements in detail and provided technical and strategical consultation on best practices. Provided ongoing support to the client throughout implementation and achieved a high satisfaction level. Manage & foster strong ongoing client relationship & provide support to existing clients.
+
+## Education & Academic Projects
+
+**University of Toronto (2013-2019)**, Honours Bachelor of Science with Distinction (B.Sc.), Computer Science
+
+
+- **Course Scheduling Website:** Created a website with algorithms that generate conflict-free course schedules based on user input. Coded front-end in HTML, CSS, JavaScript, JQuery, and Ajax; with back-end in Node.js server and SQLite database.
```
Comment (jarviscanada) [inline README.md]: Developed a website

File: README.md [hunk 6]
```diff
@@ -1,4 +1,49 @@
...
+
+- **Face-tracking Cursor:** Created an app that allows users to control the cursor using a webcam. Coded in Python with OpenCV & PyAutoGUI to calibrate face and track real-time nose position to move the cursor. Built a chrome extension that introduces new on-screen functions for better accessibility.
```
Comment (jarviscanada) [inline README.md]: Designed and implemented an app

**Review by jarviscanada — APPROVED** [review]

**Review by jarviscanada — APPROVED** [review]

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**